GameDataParser/Parsers/ItemParser.cs

```diff
@@ -16,8 +16,56 @@ public class ItemParser : Exporter<List<ItemMetadata>>
 
         protected override List<ItemMetadata> Parse()
         {
+
+            Dictionary<long, List<ItemContent>> itemDrops = new Dictionary<long, List<ItemContent>>();
+            foreach (PackFileEntry entry in resources.xmlFiles) // First,parse all boxes rewards
+            {
+                if (!entry.Name.StartsWith("table/individualitemdrop") && !entry.Name.StartsWith("table/na/individualitemdrop"))
+                {
+                    continue;
+                }
+
+                XmlDocument innerDocument = resources.xmlMemFile.GetDocument(entry.FileHeader);
+                XmlNodeList individualBoxItems = innerDocument.SelectNodes($"/ms2/individualDropBox");
+                foreach (XmlNode individualBoxItem in individualBoxItems)
+                {
+                    // Skip locales other than NA in table/na
+                    if (entry.Name.StartsWith("table/na/individualitemdrop") && individualBoxItem.Attributes["locale"] != null)
+                    {
+                        if (!individualBoxItem.Attributes["locale"].Value.Equals("NA"))
+                        {
+                            continue;
+                        }
+                    }
+
+                    long box = long.Parse(individualBoxItem.Attributes["individualDropBoxID"].Value);
+                    int id = int.Parse(individualBoxItem.Attributes["item"].Value);
+
+                    if (individualBoxItem.Attributes["minCount"].Value.Contains("."))
+                    {
+                        continue;
+                    }
+                    int minAmount = int.Parse(individualBoxItem.Attributes["minCount"].Value);
+                    int maxAmount = int.Parse(individualBoxItem.Attributes["maxCount"].Value);
+                    int dropGroup = int.Parse(individualBoxItem.Attributes["dropGroup"].Value);
+                    int smartDropRate = string.IsNullOrEmpty(individualBoxItem.Attributes["smartDropRate"]?.Value) ? 0 : int.Parse(individualBoxItem.Attributes["smartDropRate"].Value);
+                    int contentRarity = string.IsNullOrEmpty(individualBoxItem.Attributes["PackageUIShowGrade"]?.Value) ? 0 : int.Parse(individualBoxItem.Attributes["PackageUIShowGrade"].Value);
+                    int enchant = string.IsNullOrEmpty(individualBoxItem.Attributes["enchantLevel"]?.Value) ? 0 : int.Parse(individualBoxItem.Attributes["enchantLevel"].Value);
+                    int id2 = string.IsNullOrEmpty(individualBoxItem.Attributes["item2"]?.Value) ? 0 : int.Parse(individualBoxItem.Attributes["item2"].Value);
+
+                    if (itemDrops.ContainsKey(box))
+                    {
+                        itemDrops[box].Add(new ItemContent(id, minAmount, maxAmount, dropGroup, smartDropRate, contentRarity, enchant, id2));
```

Comment:
Create the instance of ItemContent before the if statement, then use it here and on line 62